src/views/auth/login-facebook-callback.pug

```diff
@@ -0,0 +1,103 @@
+doctype html
+html(lang='en')
+    head
+        title FB Login Callback - Smart House
+        script.
+            (function (window) {
```

Comment:
Not tested :) I don't know how to exclude this JS from template and import with <script> tag. Because all static code on front-end side (SmartHouse-frontend)